lib/round.rb

```diff
@@ -0,0 +1,86 @@
+class Round
+  attr_reader :deck, :turns, :current_card, :number_correct
+
+  def initialize(deck)
+    @deck = deck
+    @turns = []
+    @number_correct = 0
+
+  end
+
+  def current_card
+     deck.cards[0]
+  end
+
+
+
+  def take_turn(new_guess)
+     turn = Turn.new(new_guess, current_card)
+     @turns << turn
+     if new_guess == current_card.answer
+       @number_correct += 1
+     end
+     deck.cards.rotate!
+     return turn
+   end
+
+   def number_correct_by_category(category)
+     number_correct_by_category = 0
+
+     @turns.each do |turn|
+       if turn.card.category == category && turn.correct?
+         number_correct_by_category += 1
+       end
+     end
+     return number_correct_by_category
+   end
+
+   def percent_correct
+     percent = (number_correct.to_f / turns.count) *100
+     return percent
```

Comment:
I notice you're using a lot of explicit `return` statements, which is fine - try to get used to using the "implicit" return as well, it tends to make code a lot cleaner overall.